deployment/terraform-ansible/deploy-pulsar.yaml

```diff
@@ -123,12 +124,45 @@
   connection: ssh
   become: true
   tasks:
-    - name: Download Pulsar IO package
-      unarchive:
-        src: http://archive.apache.org/dist/incubator/pulsar/pulsar-{{ pulsar_version }}/apache-pulsar-io-connectors-{{ pulsar_version }}-bin.tar.gz
-        remote_src: yes
-        dest: /opt/pulsar
-        extra_opts: ["--strip-components=1"]
+    - name: Create connectors directory
+      file:
+        path: "/opt/pulsar/{{ item }}"
+        state: directory
+      loop:
+        - connectors
+    - name: Download Pulsar IO packages
+      get_url:
+        url: https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=pulsar/pulsar-{{ pulsar_version }}/connectors/pulsar-io-{{ item }}-{{ pulsar_version }}.nar
+        dest: /opt/pulsar/connectors/pulsar-io-{{ item }}-{{ pulsar_version }}.nar
+      loop:
+#        - aerospike
```

Comment:
Should these go uncommented by default, to keep it consistent with previous behavior?

Reply:
I'm not sure that the previous setup was widely used, given how out of date it was. I feel like it's more of a tutorial / jumping off point, than something that someone would use regularly and be confused by its behavior.

It's just a lot of time to download all of them.  Feels a bit more friendly to only download a couple, given that these are optional and that people using this are probably trying it out for the first time.

Reply:
I added a note in the documentation page about the connectors